web/res/php/webtemplate.php

```diff
@@ -27,28 +27,229 @@ public function head() {?>
   <title><?php echo $this->t;?></title>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css">
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.3/css/bootstrap.min.css" integrity="sha384-MIwDKRSSImVFAZCVLtU0LMDdON6KVCrZHyVQQj6e8wIEJkW4tvwqXrbMIya1vriY" crossorigin="anonymous">
+  <!-- CUSTOM THEMES HERE!!! -->
```

Comment:
Sorry, didn't know u were using Bootstrap 4.0.0 or that I was using an old version of the themes. My bad!